src/main/groovy/com/matthewprenger/cursegradle/CurseArtifact.groovy

```diff
@@ -61,6 +61,11 @@ class CurseArtifact implements Serializable {
     @SerializedName("relations")
     CurseRelation curseRelations
 
+    /**
+     * The curseforge file ID of the artifact.
+     */
+    Integer fileID
```

Comment:
```suggestion
    transient Integer fileID
```
Can you make the field `transient` (like the others above) so it's not serialized to the Json? The SerializedName annotations are really just to be verbose. Without them, this field would still be serialized.